docs/userGuide/syntax/footnotes.mbdf

```diff
@@ -0,0 +1,43 @@
+<span id="main-example-markbind">
```

Comment:
Remember to add the name of this file to the relevent nunjucks variable so that this documentation appears in the right places of the website.